mscore/inspector/inspector_hairpin.ui

```diff
@@ -293,6 +293,20 @@
        </property>
       </widget>
      </item>
+     <item row="2" column="0">
+      <widget class="QCheckBox" name="hairpinSingleNote">
+       <property name="text">
+        <string>Single note (de)crescendo</string>
```

Comment:
I think that might be confusing as will, since it will also change something for every single note if the hairpin spans multiple notes. Maybe something like "change volume during note"?